### PR TITLE
EHN: adding greater_equal this paddle tensor.py

### LINKED ISSUE #17745: greater_equal
adding greater_equal in paddle.tensor.tesnor.tesnor

### PR DESCRIPTION
**Pull request by MuhammadNizamani**
adding  greater_equal from issue Closed #17745

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Review by zaeemansari70 — COMMENTED** [review]
Hi @MuhammadNizamani ,

The PR looks great and it's ready to merge, kindly resolve your conflicts on this PR and let me know so I can merge it.

Thanks! :slightly_smiling_face:

**Review by zaeemansari70 — COMMENTED** [review]
Hi @MuhammadNizamani  
The conflicts are still there :slightly_smiling_face:

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 1]
```diff
@@ -2974,3 +2974,34 @@ def test_paddle_atanh(
...
+    method_name="greater_equal",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"), num_arrays=2, shared_dtype=True
+    ),
+)
+def test_paddle_greater_equal(
+    dtypes_and_x,
+    frontend_method_data,
+    init_flags,
+    method_flags,
```
Comment (zaeemansari70) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2992]: backend_fw

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 2]
```diff
@@ -2974,3 +2974,34 @@ def test_paddle_atanh(
...
+    frontend,
+    on_device,
+):
+    input_dtype, x = dtypes_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        init_all_as_kwargs_np={"data": x[0]},
+        method_input_dtypes=input_dtype,
```
Comment (zaeemansari70) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:3000]: backend_to_test=backend_fw

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
A small change is needed. there has been some changes in the testing pipeline, I have added the changes that you'll need.

I've locally tested your code, after that, you will probably need to add the decorator to handle the dtype errors you'll face.
Also, kindly reslove the merge conflicts as they seem to appear again :sweat_smile:  

Thanks!

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
It's failing for the following error:
```
E           ivy.utils.exceptions.IvyValueError: numpy: nested_map: numpy: nested_map: (InvalidArgument) Input object type error or incompatible array data type. tensor.set() supports array with bool, float16, float32, float64, int8, int16, int32, int64, uint8 or uint16, please check your input or input array data type. (at /paddle/paddle/fluid/pybind/tensor_py.h:536)
E           
E           Falsifying example: test_paddle_greater_equal(
E               backend_fw='tensorflow',
E               frontend='paddle',
E               on_device='cpu',
E               dtypes_and_x=(['bfloat16', 'bfloat16'],
E                [array(-1, dtype=bfloat16), array(-1, dtype=bfloat16)]),
E               frontend_method_data=FrontendMethodData(ivy_init_module='ivy.functional.frontends.paddle', framework_init_module='paddle', init_name='to_tensor', method_name='greater_equal'),
E               method_flags=FrontendMethodTestFlags(
E                   num_positional_args=0,
E                   as_variable=[False],
E                   native_arrays=[False],
E               ),
E               init_flags=FrontendMethodTestFlags(
E                   num_positional_args=1,
E                   as_variable=[False],
E                   native_arrays=[False],
E               ),
E           )
E           
E           You can reproduce this example by temporarily adding @reproduce_failure('6.82.0', b'AXicY2YAAkYGCGBEY4MBAADUAAg=') as a decorator on your test case
```
"bfloat16" is not supported, add that in the unsupported dtypes.
Kindly request a review once the tests are passing.
Feel free to ask any question by commenting!
Thanks!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by MuhammadNizamani** [comment]
I am working on it

**Comment by MuhammadNizamani** [comment]
I am working on this

**Comment by MahmoudAshraf97** [comment]
Hi Muhammad, Feel free to open a new PR when it's ready

**Pull request closed without merging**